sources/packages/api/src-sql/Reports/Create-disbursement-report.sql

```diff
@@ -0,0 +1,44 @@
+--Insert config for disbursement report.
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Disbursement_Report',
+        'SELECT "Date of Disbursement",
```

Comment:
Was this query formatted with the same formater that we use in VSCode. If I format it on my VS Code it looks different.

Reply:
We can connect, it is not working for me

Reply:
As discussed changed